Packs/FreshworksFreshservice/Integrations/FreshworksFreshservice/FreshworksFreshservice.py

```diff
@@ -0,0 +1,3630 @@
+
+            # Convert str value to int in case it's a predefined values
+            if entity_name in ['ticket', 'problem', 'change', 'release']:
+                if dict_key := TICKET_PROPERTIES_BY_TYPE[entity_name].get(key):
+                    integer_value = dict_key.get(value)
+                    correct_value = integer_value if integer_value is not None else correct_value
+
+            if updated_query:
+                updated_query += f" AND {key}:{correct_value}"
+            else:
+                updated_query = f"{key}:{correct_value}"
+
+    if updated_query:
+        updated_query = f'"{updated_query}"'
+
+    return updated_query
+
+
+def convert_date_time(date_time: str) -> str | None:
+    """ Convert str to datetime.
+
+    Args:
+        date_time (str): The datetime str.
+
+    Returns:
+        str: The updated datetime.
+    """
+    datetime_arg = arg_to_datetime(date_time)
+    if isinstance(datetime_arg, datetime):
+        updated_datetime_arg = datetime_arg.strftime(STRFTIME)
+    return updated_datetime_arg
+
+
+def validate_pagination_arguments(
+    page: Optional[int] = None,
+    page_size: Optional[int] = None,
+    limit: Optional[int] = None,
+):
+    """ Validate pagination arguments according to their default.
+
+    Args:
+        page (int, optional): Page number of paginated results.
+        page_size (int, optional): Number of items per page.
+        limit (int, optional): The maximum number of records to retrieve.
+
+    Raises:
+        ValueError: Appropriate error message.
+    """
+    if page_size and (page_size < MIN_PAGE_SIZE or page_size > MAX_PAGE_SIZE):
+        raise ValueError(
+            f"page size argument must be greater than {MIN_PAGE_SIZE} and smaller than {MAX_PAGE_SIZE}."
+        )
+    if page is not None and page < MIN_PAGE_NUM:
+        raise ValueError(
+            f"page argument must be greater than {MIN_PAGE_NUM-1}.")
+    if limit is not None and limit <= MIN_LIMIT:
+        raise ValueError(f"limit argument must be greater than {MIN_LIMIT}.")
+
+
+def pagination(args: Dict[str, Any]) -> Tuple:
+    """ Return the correct limit and offset for the API
+        based on the user arguments page, page_size and limit.
+
+    Args:
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        Tuple: new_limit, offset, pagination_message.
+    """
+    page = arg_to_number(args.get("page"))
+    page_size = arg_to_number(args.get("page_size"))
+    limit = arg_to_number(args.get("limit"))
+
+    validate_pagination_arguments(page, page_size, limit)
+
+    new_page = 1
+    new_page_size = limit
+
+    if page is not None and page_size:
+        new_page_size = page_size
+        new_page = page
+
+    pagination_message = f"Showing page {new_page}. \n Current page size: {new_page_size}."
+
+    return new_page, new_page_size, pagination_message
+
+
+def date_to_epoch_for_fetch(date: Optional[datetime]) -> int:
```

Comment:
We don't need the function for one line

Reply:
Same as before, but why would we want to repeat a logic multiple times? Also, it will be easier to read and maintain if any changes are needed.